### ISSUE TITLE
[BUG] [DEV] ACF Armor Properties tool breaks when used with spherical props

### ISSUE THREAD

**Issue by CheezusChrust** [Stooberton/ACF-3#85]
**Short Description**

On the latest version of the dev branch, when setting or viewing the armor of an entity that's been made spherical, an error continually gets spewed through the console and the tooltip reports the armor as "NaN".

**Console Error Logs (Optional)**

```
addons/acf-3-dev/lua/acf/base/sv_validation.lua:157: attempt to perform arithmetic on a nil value
  1. ACF_Activate - addons/acf-3-dev/lua/acf/base/sv_validation.lua:157
   2. ACF_Check - addons/acf-3-dev/lua/acf/base/sv_validation.lua:137
    3. Think - addons/acf-3-dev/lua/weapons/gmod_tool/stools/acfarmorprop.lua:184
     4. unknown - gamemodes/sandbox/entities/weapons/gmod_tool/shared.lua:187 (x1950)
```

**How to Reproduce (Optional)**

Apply armor to any spherical prop. May take a few attempts.

**Screenshots (Optional)**

![gmod_oNkfR7OpQ0](https://user-images.githubusercontent.com/21130637/88358599-91245d00-cd3d-11ea-991b-54b1d335318f.png)

**Additional Information (Optional)**

This error also appears in chat upon spawning a dupe containing the bugged spherical entities.
```
Error applying entity modifer, "acfsettings".
ERROR: addons/acf-3-dev/lua/acf/base/sv_validation.lua:157: attempt to perform arithmetic on a nil value
```

**Comment by CheezusChrust** [comment]
Messing around more and this seems to entirely break ballistics when anything spherical gets hit.

**Comment by TwistedTail** [comment]
Fixed on 5df146a.
